draft-ietf-avtcore-rtp-over-quic.md

```diff
@@ -1270,20 +1274,51 @@ The "rtp-mux-quic" string identifies RoQ:
   Specification:
   : This document
 
-## RoQ Error Codes
+## RoQ Error Codes Registry {#iana-error-codes}
+
+This document establishes a registry for RoQ error codes. The "RoQ Error Codes"
+registry manages a 62-bit space.
+
+The new error codes registry created in this document operates under the QUIC
+registration policy documented in {{Section 22.1 of !RFC9000}}. This registry
+includes the common set of fields listed in {{Section 22.1.1 of !RFC9000}}.
+
+Permanent registrations in this registry are assigned using the Specification
+Required policy ({{!RFC8126}}), except for values between 0x00 and 0x3f (in
+hexadecimal; inclusive), which are assigned using Standards Action or IESG
+Approval as defined in {{Sections 4.9 and 4.10 of !RFC8126}}.
+
+Registrations for error codes are required to include a description of the error
+code. An expert reviewer is advised to examine new registrations for possible
+duplication with existing error codes.
+
+In addition to common fields as described in Section {{Section 22.1 of
+!RFC9000}}, this registry includes two additional fields. Permanent
+registrations in this registry MUST include the following fields:
+
+Name:
+: A name for the error code.
+
+Description:
+: A brief description of the error code semantics.
+
+The initial allocations in this registry are all assigned permanent status and
+list a change controller of the IETF and a contact of the AVTCORE working group
+(avt@ietf.org).
 
-This document establishes a registry for RoQ error codes.
+The entries in {{tab-error-codes}} are registered by this document.
```

Comment:
Especially because we're tying this registry to QUIC practice, the initial registration should probably contain all the fields for each entry, in the order we want them to appear, as the QUIC registries [here](https://www.iana.org/assignments/quic/quic.xhtml#quic-transport-error-codes) show.

Reply:
I think all fields are included: *Name*, *Value*, *Description*, and *Specification* are in the table, *Status*, *Change Controller*, and *Contact* for all are given in the text above. HTTP/3 uses this order but the table in the registry uses a different order. I'll swap it here, so we may have the same order here and in the registry.

Reply:
As we discussed yesterday, I knew we were trying to match the QUIC error codes, but didn't realize that HTTP/3 error codes and DNS over QUIC error codes didn't match that order, or each other. My apologies, obviously my comment doesn't matter!